tools/postprocess/deserialize_meta.py

```diff
@@ -0,0 +1,63 @@
+import os
+import json
+import pathlib
+import jsonlines
+from multiprocessing import Pool
+import fire
+
+
+def fp_iter(src_dir):
+    """
+    Find all jsonl files in the source directory.
+    :param src_dir: path to source dataset directory
+    :return: iterator over jsonl files
+    """
+    for fp in pathlib.Path(src_dir).glob('*.jsonl'):
+        yield fp
+
+
+def meta_deserialize(file_name, target_file, serialized_key):
+    """
+    Unerialize the specified field into dict.
+    :param file_name: path to source jsonl files.
+    :param target_file: path to store the converted jsonl files.
+    :param serialized_key: the key corresponding to the field that will be
+    deserialized.
+    """
+    with open(target_file, 'w') as fw:
+        with jsonlines.open(file_name, 'r') as fr:
+            for obj in fr:
+                obj[serialized_key] = json.loads(obj[serialized_key])
+                fw.write(json.dumps(obj, ensure_ascii=False) + '\n')
```

Comment:
Using these tools, an object is serialized and deserialized like below:
```text
# before serializing
{
  'text': 'xxx',
  'meta1': 'val1',
  'meta2': 'val2',
}
# after serializing
{
  'text': 'xxx',
  'serialize_key': '{"meta1": "val1", "meta2": "val2"}'
}
# after deserializing
{
  'text': 'xxx',
  'serialize_key': {
    'meta1': 'val1',
    'meta2': 'val2'
  }
}
```
The deserialized object is inconsistent with the original object.